ffka-gluon-web-usb-wan-hotplug/i18n/de.po

```diff
@@ -18,7 +18,8 @@ msgstr "USB-WAN-Hotplug"
 
 msgid "ffka-gluon-web-usb-wan-hotplug:description"
 msgstr ""
-"Dein Knoten kann automatisch einen per USB angeschlossen Ethernet-Adapter als WAN benutzen. "
-"Das ist insbesondere Praktisch wenn als WAN der mobile Hotspot eines per USB verbundenen "
-"Mobiltelefons verwendet werden soll. Für iOS wird automatisch die Trust-Frage ausgelöst, "
-"wenn diese positiv beantwortet wird stellt auch iOS ein Ethernet Gerät über USB zur verfügung."
+"Dein Knoten kann automatisch einen per USB angeschlossen Ethernet-Adapter als "
+"WAN benutzen. Das ist insbesondere Praktisch wenn als WAN ein per USB "
+"verbundenes Mobiltelefons verwendet werden soll, das USB-Tethering aktiviert "
+"hat. Für iOS wird automatisch die Trust-Frage ausgelöst, wenn diese positiv "
```

Comment:
I disagree, activating USB tethering is part of the process as outlined  in the following sentence. If you refer to the support of the client device, this would make more sense to me.

Reply:
Ich wollte hier nur klar stellen, dass es darum geht, ein Mobiltelefon direkt per USB anzuschliessen. Vorher klang es in dem deutschen Text so, als wenn man einen USB-WLAN-Adapter-Dongle anschliessen solle, was nicht viel sinn ergab.

Ich verstehe nicht ganz, was du meinst, was an der Formulierung jetzt unlogisch sei. Vielleicht kannst du es einmal anders Formulieren, wie es besser klingt?

Reply:
The sentence "Das ist insbesondere Praktisch wenn als WAN der mobile Hotspot eines per USB verbundenen  Mobiltelefons verwendet werden soll.". Activating the function is part of the process and as such already implied in the description already there.

Splitting the sentence to describe the process distinctly makes the description harder to read in my opinion. If your intention was to highlight support is required from the client device side, this is better done explicitly.

Reply:
Vielleicht formulierst du den deutschen text einfach mal eben, wie du ihn logisch finden würdest. Denke das wäre am einfachsten im Moment um das zu klären. Der en.po -Text war schon eindeutig und bedurfte keiner Änderung.

Reply:
Wenn ich das hier nochmal lese:
ich finde die Änderung von @rubo77 sinnvoll. Das Wort "USB-Tethering" ist hier wichtig und kam vorher nicht in der Beschreibung vor.
Ich habe das jetzt mal rebased und aktualisiert. Hoffe das ist für alle so okay :)